superset/assets/spec/javascripts/dashboard/components/HeaderActionsDropdown_spec.jsx

```diff
@@ -105,9 +105,9 @@ describe('HeaderActionsDropdown', () => {
       expect(wrapper.find(MenuItem)).toHaveLength(2);
     });
 
-    it('should render the RefreshIntervalModal', () => {
+    it('should not render the RefreshIntervalModal', () => {
```

Comment:
should we add tests that assert it renders this in `editMode`?

Reply:
We have them already, actually. I didn't have to change them. See https://github.com/apache/incubator-superset/pull/7248/files#diff-c130332fd863f354e0dd6e8e6097f72fR142